src/canvas/PathIntersection.tsx

```diff
@@ -49,16 +48,12 @@ export class PathIntersection {
         return null;
     };
 
-    /* Finds if the start point of a path intersects with any other path, and if so, returns that intersection point.*/
-    static findPathStartIntersectAlongPatternPath = (path: PatternPath, paths: PatternPath[]): IIntersection | null => {
+    /* Finds if the a point intersects with any other path, and if so, returns that intersection point.*/
```

Comment:
Type in comment "Finds if the a point" should be "Finds if a point"

Reply:
oops. done.